mlflow/entities/run_info.py

```diff
@@ -8,25 +8,23 @@
 def check_run_is_active(run_info):
     if run_info.lifecycle_stage != LifecycleStage.ACTIVE:
         raise MlflowException("The run {} must be in 'active' lifecycle_stage."
-                              .format(run_info.run_uuid))
+                              .format(run_info.run_id))
 
 
 def check_run_is_deleted(run_info):
     if run_info.lifecycle_stage != LifecycleStage.DELETED:
         raise MlflowException("The run {} must be in 'deleted' lifecycle_stage."
-                              .format(run_info.run_uuid))
+                              .format(run_info.run_id))
 
 
 class RunInfo(_MLflowObject):
     """
     Metadata about a run.
     """
 
-    def __init__(self, run_uuid, experiment_id, name, source_type, source_name, entry_point_name,
-                 user_id, status, start_time, end_time, source_version, lifecycle_stage,
-                 artifact_uri=None):
-        if run_uuid is None:
-            raise Exception("run_uuid cannot be None")
+    def __init__(self, run_uuid, experiment_id, name, source_type, source_name,
```

Comment:
This constructor lgtm, makes sense to keep `run_uuid` in the constructor so we can still deserialize run data persisted to the FileStore in older versions via `_MlflowObject.from_dictionary` (and  for `run_id` to be optional for the same reason)